app/views/helpers/_top_project.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="col s4">
```

Comment:
@lukyans I moved the "top project" card/view to a partial, so it'll only be rendered IF there is a top project.

So, when you've got the method working, you'll need to mess around with the card here to get it to display.

Let me know if you have questions about this!